### ISSUE TITLE
download failure

### ISSUE THREAD

**Issue by syncrholord** [ssaunderss/clinvar-checker#5]
running clinvar_checker_linux on fedora. fails to download clinvar database

```sh
./clinvar_checker_linux download
Downloading ClinVar data...


19:59:32.245 [notice] Application clinvar_checker exited: exited in: ClinvarChecker.Application.start(:normal, [])
    ** (EXIT) an exception was raised:
        ** (File.Error) could not write to file "tmp/clinvar.vcf.gz": no such file or directory
            (elixir 1.17.3) lib/file.ex:1144: File.write!/3
            (clinvar_checker 1.1.0) lib/clinvar_checker.ex:61: ClinvarChecker.download_clinvar_data/0
            (clinvar_checker 1.1.0) lib/clinvar_checker/cli.ex:30: ClinvarChecker.Cli.parse_command/1
            (clinvar_checker 1.1.0) lib/clinvar_checker/application.ex:19: ClinvarChecker.Application.start/2
            (kernel 10.1.1) application_master.erl:295: :application_master.start_it_old/4
Kernel pid terminated (application_controller) ("{application_start_failure,clinvar_checker,{bad_return,{{'Elixir.ClinvarChecker.Application',start,[normal,[]]},{'EXIT',{#{reason => enoent,path => <<\"tmp/clinvar.vcf.gz\">>,'__struct__' => 'Elixir.File.Error','__exception__' => true,action => <<\"write to file\">>},[{'Elixir.File','write!',3,[{file,\"lib/file.ex\"},{line,1144}]},{'Elixir.ClinvarChecker',download_clinvar_data,0,[{file,\"lib/clinvar_checker.ex\"},{line,61}]},{'Elixir.ClinvarChecker.Cli',parse_command,1,[{file,\"lib/clinvar_checker/cli.ex\"},{line,30}]},{'Elixir.ClinvarChecker.Application',start,2,[{file,\"lib/clinvar_checker/application.ex\"},{line,19}]},{application_master,start_it_old,4,[{file,\"application_master.erl\"},{line,295}]}]}}}}}")

Crash dump is being written to: erl_crash.dump...done
```

**Comment by ssaunderss** [comment]
@syncrholord - Thanks for bringing this to my attention. It looks like it's complaining that the `tmp` directory doesn't exist so you might need to manually create it from where you are storing/running the CLI via `mkdir tmp`. Then if you run the script again it should be able to download the data and run.

I'll work on a fix for this, but please let me know if that works or you run into any other issues in the meantime!
